apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/Profile.java

```diff
@@ -0,0 +1,88 @@
+package gov.cms.bfd.server.war.commons;
+
+import static gov.cms.bfd.server.war.commons.ProfileConstants.C4BB_VERSION_SUFFIX;
+import static gov.cms.bfd.server.war.commons.ProfileConstants.C4DIC_VERSION_SUFFIX;
+import static java.util.Objects.requireNonNull;
+
+import java.util.EnumSet;
+
+/** Represents the supported CARIN profiles. */
+public enum Profile {
+
+  /** Represents the CARIN Blue Button Implementation Guide. */
```

Comment:
@aschey-forpeople should this also include the EOB URL?

Reply:
I was considering that. I left it out because there's no C4DIC version of it but I can suppose we can set it to null,

Reply:
Done, added all the URLs in here for consistency.